website/src/mode.tsx

```diff
@@ -225,3 +236,43 @@ const TagSetOption = (p: { radio: RadioStateReturn; level: string }) => {
     </Tooltip>
   )
 }
+
+export const PrefPanel = () => {
+  const preferences = useContext(preferencesContext)
+  return (
+    <div>
+      <h3>Display Mode:&ensp; </h3>
+      {<ExperiencePicker onSelect={preferences.expLevelUpdate} />}
+      <p>{levelNameMapping[preferences.expLevel].details}</p>
+      <h3>Romanization Method:&ensp; </h3>
+      {<PhoneticsPicker onSelect={preferences.phonRepUpdate} />}
+      <p>{phoneticRepresentationMapping[preferences.phonRep].details}</p>
```

Comment:
Could you look into how we might be able to semantically connect the picker to this description? Maybe `aria-describedby` would do the trick.

Reply:
Do you mean just showing better how the desccriptions correspond to the selectors within the code? Or are there functional changes as well. I don't really understand what it means to semantically connect them.